platform/viewer/cypress/integration/OHIFStudyViewer.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('OHIF Study Viewer Page', () => {
+  
+  before(() => {
+      // cy.visit('/');
+      // cy.get('#patientName')
+      //   .type("Dummy");
+      // cy.get('.studylistStudy > .patientName')
+      //   .contains("Dummy")
+      //   .click();
+      cy.openStudy("MISTER^MR");
+    });
+  
+    it('checks if series thumbnails are being displayed', ()=> {
+      cy.get('.ThumbnailEntryContainer')
+      .its('length')
+      .should('be.gt', 1);
+    });
+
+    it('drags and drop a series thumbnail into viewport', () => {
+      cy.wait(3000); 
+      cy.get('.ThumbnailEntryContainer:nth-child(2)')
+        .drag('.cornerstone-canvas');
+      cy.wait(1000); 
```

Comment:
Ideally, we don't need to sprinkle in `wait`'s. I'll make sure the issue you found is addressed so we can avoid this anti-pattern.